public/stylesheets/style.css

```diff
@@ -42,38 +42,20 @@ a {
   margin-bottom: 20px;
 }
 
-#subProject {
+#btnSubProject {
   margin-bottom: 20px;
   margin-left: 10px;
 }
 
-#btnDownload {
-  display: none;
-  width: -moz-fit-content;
-  margin-right: 10px;
-}
-
-#btnPreview {
+#btnDeployHeroku, #btnDeployGithub, #btnPreview, #btnDownload {
   display: none;
   width: -moz-fit-content;
   margin-left: 10px;
 }
 
-#btnLogs {
-  display: none;
-  width: -moz-fit-content;
-}
-
-#btnDeployGithub {
+#btnWebsiteLink, #btnGithubPages, #btnLogs {
```

Comment:
Cannot see a logical classification for these buttons. I don't think there is a need for that